liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/InstallFeatureSupport.java

```diff
@@ -132,14 +134,16 @@ protected boolean initialize() throws MojoExecutionException {
         return true;
     }
 
-    protected Set<String> getInstalledFeatures() throws PluginExecutionException {
+    protected Set<String> getInstalledFeatures(InstallFeatureUtil util) throws PluginExecutionException {
```

Comment:
Add a javadoc comment for this method particularly since it now accepts a parameter.